src/pages/ProfilePage.jsx

```diff
@@ -2,9 +2,10 @@ import React from "react";
 
 class ProfilePage extends React.Component {
   renderAbout(name) {
-    const obj = {
+    var obj = {
```

Comment:
Meu querido, qual o motivo do VAR?
```suggestion
    const obj = {
```

Reply:
VAR é mais bonitinho.